api/docker/php/migrate-database.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+set -e
+
+echo "Waiting for database to be ready..."
+ATTEMPTS_LEFT_TO_REACH_DATABASE=60
+until [ $ATTEMPTS_LEFT_TO_REACH_DATABASE -eq 0 ] || DATABASE_ERROR=$(php bin/console dbal:run-sql -q "SELECT 1" 2>&1); do
+  if [ $? -eq 255 ]; then
+    # If the Doctrine command exits with 255, an unrecoverable error occurred
+    ATTEMPTS_LEFT_TO_REACH_DATABASE=0
+    break
+  fi
+  sleep 1
+  ATTEMPTS_LEFT_TO_REACH_DATABASE=$((ATTEMPTS_LEFT_TO_REACH_DATABASE - 1))
+  echo "Still waiting for database to be ready... Or maybe the database is not reachable. $ATTEMPTS_LEFT_TO_REACH_DATABASE attempts left."
+done
+
+if [ $ATTEMPTS_LEFT_TO_REACH_DATABASE -eq 0 ]; then
+  echo "The database is not up or not reachable:"
+  echo "$DATABASE_ERROR"
+  exit 1
+else
+  echo "The database is now ready and reachable"
+fi
+
+if ls -A migrations/*.php >/dev/null 2>&1; then
+  php bin/console doctrine:migrations:migrate --no-interaction
```

Comment:
The migrations are run in a separate pod (i.e. a php container is spun up only to run the migrations) every time we install or upgrade a release. This pod is configured in  If that pod runs into an error, it will remain in the cluster. So in there, we'd be able to see the logs.
If the pod runs smoothly, it's removed from the cluster afterwards. We could choose to leave it there, but then our cluster would slowly fill up with these finished pods from old feature branch deployments... I think when uninstalling the feature branch deployment, Helm cannot automatically remove these old pods that were created by a hook. We'd have to add another command in continuous-deployment.yml after `helm delete` to accomplish that, and it could get forgotten if we ever manually delete a feature branch deployment.